packages/core/chrome/core-chrome-browser-internal/src/chrome_service.tsx

```diff
@@ -170,6 +175,14 @@ export class ChromeService {
       chromeStyle$.next(style);
     };
 
+    const replaceProjectSideNav = (sideNav: JSX.Element) => {
```

Comment:
I wouldn't provide this method, to avoid race conditions or conflicting sets.  I would instead throw if someone tries to set it twice... unless you see a reason replacing is a good idea, (I personally don't).

Reply:
the idea is that `setProjectNavigation(configObject)` for setting our tree config, but `replaceProjectSideNav(jsxElement)` is for case when they need override our default project sidenav component.